src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -398,17 +399,17 @@ class TezosNodeOperator(val node: TezosRPCInterface, val network: String, batchC
 
   val UNPARSABLE_CODE_PLACEMENT = "Unparsable code: "
 
-  private def toMichelsonScript[T <: MichelsonElement:Parser](json: Any): String = {
+  private def toMichelsonScript[T <: MichelsonElement](json: Any)(implicit parser: Parser[T], tag: ClassTag[T]): String = {
```

Comment:
You can add more than one type bound, if you like to, or mix them

`toMichelsonScript[T <: MichelsonElement: Parser](json: Any)(implicit tag: ClassTag[T]): String`